pkg/util/taints/taints.go

```diff
@@ -94,3 +103,199 @@ func (t taintsVar) String() string {
 func (t taintsVar) Type() string {
 	return "[]api.Taint"
 }
+
+// ParseTaints takes a spec which is an array and creates slices for new taints to be added, taints to be deleted.
+func ParseTaints(spec []string) ([]v1.Taint, []v1.Taint, error) {
```

Comment:
As of now, none, I want to add them in a separate PR as this is just for moving the fn's.